Goodboy-Oldboy.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+const dogYearOfBirth = 2018;
+const dogYearFuture = 2025;
+const shouldShowResultInDogYears = false;
+const humanYears = dogYearFuture - dogYearOfBirth;
+const type = shouldShowResultInDogYears ? `dog` : `human`;
```

Comment:
Fine to use backtick here, but if there's no `${}`, common practice is to use normal quotations

Reply:
grape